bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/constants/RuntimeConstants.java

```diff
@@ -117,11 +117,10 @@
     public static final String INTERNAL_ERROR_MESSAGE =
             "ballerina: Oh no, something really went wrong.\n" +
             "\n" +
-            "The `ballerina-internal.log` file located in the current directory\n" + 
-            "will indicate what the problem is.\n" +
-            "We really appreciate it if you can share with us, the code\n" +
-            "that broke Ballerina, together with this log file by creating a\n" +
-            "bug report in https://github.com/ballerina-platform/ballerina-lang/issues.\n";
+            "If you are able to share with us the code that broke Ballerina then\n" +
+            "we would REALLY appreciate if you would report this to us:\n" +
+            "go to https://github.com/ballerina-platform/ballerina-lang/issues and\n" +
+            "create a bug report with both this log file and the sample code.\n";
```

Comment:
this should be following log

Reply:
Fixed in 59c54477e5cba0f85514920f74c271641ad31d7c